microsoft-365/security/defender-endpoint/customize-exploit-protection.md

```diff
@@ -81,10 +81,10 @@ For the associated PowerShell cmdlets for each mitigation, see the [PowerShell r
 >
 > | Enabled in **Program settings** | Enabled in **System settings** | Behavior |
 > | ------------------------------- | ------------------------------ | -------- |
-> | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | ![Check mark no](/security/defender-endpoint/images/svg/check-no) | As defined in **Program settings** |
-> | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | As defined in **Program settings** |
-> | ![Check mark no](/security/defender-endpoint/images/svg/check-no) | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | As defined in **System settings** |
-> | ![Check mark no](/security/defender-endpoint/images/svg/check-no) | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | Default as defined in **Use default** option |
+> |         Yes        |        No       | As defined in **Program settings** |
+> |         Yes        |        No       | As defined in **Program settings** |
```

Comment:
Previously Yes - Yes, are these values correct?

Reply:
Yes

Reply:
I'll check and fix this

Reply:
Fixed. Please review

Reply:
@msbemba - thank you.